middle_end/flambda2/simplify_shared/inlining_helpers.ml

```diff
@@ -115,3 +115,12 @@ let wrap_inlined_body_for_exn_extra_args acc ~extra_args ~apply_exn_continuation
   in
   let_cont_create acc wrapper ~handler_params:wrapper_handler_params
     ~handler:wrapper_handler ~body:body_with_push ~is_exn_handler:true
+
+type attribute_kind =
+  | Inlined
+  | Unroll
+
+let string_of_kind = function Inlined -> "[@inlined]" | Unroll -> "[@unroll]]"
```

Comment:
```suggestion
let string_of_kind = function Inlined -> "[@inlined]" | Unroll -> "[@unrolled]]"
```

Only the `[@unrolled <n>]` annotation exists (cf the flambda manual : https://v2.ocaml.org/manual/flambda.html#s%3Aflambda-manual-control ).